packages/sdk-react-native/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/sdk-react-native
 
+## 1.0.6
+
+### Patch Changes
+
+- e8bc05b: Introduces handleGoogleOAuth(): You can now easily handle Google OAuth authentication flow.
```

Comment:
I think this can be a minor version bump (`1.1.0`)

Can we include some usage details as well? Can be as simple as a summary of the inline documentation: https://github.com/tkhq/sdk/blob/ef702dd338f570869f5ff002f2ee71fcc99e0c40/packages/sdk-react-native/src/contexts/TurnkeyContext.tsx#L880-L895